src/plugins/wizard/public/application/components/top_nav.tsx

```diff
@@ -65,6 +69,7 @@ export const TopNav = () => {
         config={config}
         setMenuMountPoint={setHeaderActionMenu}
         indexPatterns={indexPattern ? [indexPattern] : []}
+        showDatePicker={!!indexPattern?.timeFieldName ?? true}
```

Comment:
nit - with the boolean cast, do you still need nullish coalescing?

Reply:
good call. will fix in followup :)